unit_tests/testSearch/testSearch.f90

```diff
@@ -48,150 +48,150 @@ PROGRAM testSearch
 !-------------------------------------------------------------------------------
 SUBROUTINE testLowerBound_i()
 
-  ASSERT(lowerBound(test_sorted, 0) == 1, "Wrong result for 0")
+  ASSERT(lowerBound(increasing_i, 0) == 1, "Wrong result for 0")
```

Comment:
Also, since you're in here, go ahead and switch the ASSERT/FINFO to the proper form of ASSERT_EQ

Reply:
Done